### ISSUE TITLE
Lots of requests without problems?

### ISSUE THREAD

**Issue by TravelTrader** [petercerno/good-morning#22]
I have more a question than an issue:

1. `good_morning` is the holy grail in my search for getting data from morningstar!

2. I just use it for 1-5 stocks per day. And not at the same time at the moment.

But did you really use your demo of getting the data for 500 stocks of the S&P500?
Isn't there a problem with requesting so much data?
Or would you recommend adding a proxy server option?

Thanks for your opinions!

**Comment by petercerno** [comment]
Hi,

thanks your interest. I really appreciate that this old project is still
useful to you. (I no longer have capacity to maintain it.)

If I remember correctly, I was able to get the data for 500 stocks by
requesting them in a sequence with reasonable (random) delay. You need to
try and see what works best for you.

Cheers,

Peter

**Comment by TravelTrader** [comment]
Yes, of course, it's very useful. I tried around before with the url from the source code but didn't get a good result. Therefore: It worked immediately. Thanks.

And with the requests you would say: Just do it until any problems happen?

**Comment by petercerno** [comment]
I think that you can try. In the worst case you get throttled / or
blacklisted for an hour / day. Then you can slow down or increase the
waiting intervals.

**Comment by jjphung** [comment]
@TravelTrader this repo still works for you? I tried the basic example on the Readme, but it throws an error: `MorningStar cannot find the ticker symbol you entered or it is INVALID. Please try again.` I tried messing with the url in `good_morning` for a couple of hours, but to no avail.
